From: Andrea Bolognani <eof@kiyuko.org>
Date: Wed, 1 Feb 2012 16:24:29 +0100
Subject: Set class name

The default class name for Tk application is "Tk".

This patch sets it to "Rhinote", so the correct application name
is displayed in such places as the GNOME Shell top bar.

Forwarded: Marv Boyes <greyspace@tuxfamily.org>
---
 rhinote.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rhinote.py b/rhinote.py
index 4862022..d8a9325 100644
--- a/rhinote.py
+++ b/rhinote.py
@@ -31,7 +31,7 @@ import subprocess
 
 # the root window:
 def Rhinote():
-    r = tkinter.Tk()
+    r = tkinter.Tk(className='Rhinote')
     r.option_add('*font', '{Helvetica} 11')
     t = TextWidget(r, bg='#f9f3a9', wrap='word', undo=True)
     t.focus_set()
